### PR TITLE
[SW-1347] Move invites vuex state to composable

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

File: src/composables/invites.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    if (!isMax) {
+      await aeSdk.spend(
+        amount,
+        recipientId,
+        // @ts-ignore
+        { denomination: AE_AMOUNT_FORMATS.AE },
+      );
+    } else {
+      await aeSdk.transferFunds(
+        1,
```
Comment (pstachel) [inline src/composables/invites.ts:63]: Maybe it's worth to comment this line why are we sending `1`

**Review by pstachel — APPROVED** [review]

File: src/composables/invites.ts (hunk 1 above)
Comment (peronczyk) [reply src/composables/invites.ts:63]: @CedrikNikita can you provide some info about this?

Comment (CedrikNikita) [reply src/composables/invites.ts:63]: First argument is a percentage from 0 to 1, where 0 is 0% and 1 is 100%.

**Review by subhod-i — APPROVED** [review]
LGTM.

File: src/composables/invites.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import { STORAGE_KEYS } from '@/constants';
+import {
+  AE_AMOUNT_FORMATS,
+  AeSdk,
+  Encoded,
+  Node,
+} from '@aeternity/aepp-sdk';
```
[the remark this answers is not in the document]
Comment (peronczyk) [reply src/composables/invites.ts]: Makes sense. Fixed.

**Pull request merged**